### PR TITLE
Functionality tests for Email login are done.

### PR DESCRIPTION
**Pull request by StanislavKim1**
RBTA-19-Functionality - Email. Dominate, humiliate,  suggest.

### CODE REVIEW

File: test/data/login.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export default {
   email: 'testreacttest@gmail.com',
+  unregistratedEmail:'unregistrated@gmail.com',
```
Comment (artsenius) [inline test/data/login.js]: Why do you need this if you have random email in helpers?

File: wdio.conf.js [hunk 2]
```diff
@@ -3,20 +3,21 @@ exports.config = {
...
+        // './test/specs/reg/global-footer/design.js',
+        // './test/specs/reg/BugList/General/BugList-General.js',
+        // './test/specs/reg/registration/general.js',
+        // './test/specs/reg/registration/functionality.js',
+        // './test/specs/reg/login/error.js',
+        // './test/specs/reg/forgot-password.js',
+        // './test/specs/reg/navigationBar/general-nav-bar.js',
+        // './test/specs/reg/login/general.js',
+        './test/specs/reg/login/functionality.js',
+        // './test/specs/reg/bug-report/edit-mode-.js'
```
Comment (artsenius) [inline wdio.conf.js]: uncomment everything

File: test/helpers/helpers.js [hunk 3]
```diff
@@ -55,6 +55,20 @@ class Helpers {
...
+    randomEmail.clearValue();
+    return actual;
+  }
+  randomPassword(selector){
+    let randomPassword = $(selector);
+    randomPassword.addValue('AnyPassword');
+    let actual = randomPassword.addValue('AnyPassword');
+    randomPassword.clearValue();
+    return actual;
+  }
```
Comment (artsenius) [inline test/helpers/helpers.js]: Why you wrote those two functions if you don't use them?

Comment (artsenius) [reply test/helpers/helpers.js]: I don't see help.randomEmail() and help.randomPassword()in test files.

File: test/data/login.js (hunk 1 above)
Comment (StanislavKim1) [reply test/data/login.js]: Oh, true. That makes sense. I will fix it

File: test/helpers/helpers.js (hunk 3 above)
Comment (StanislavKim1) [reply test/helpers/helpers.js]: Ok I will fix it ASAP

File: wdio.conf.js (hunk 2 above)
Comment (StanislavKim1) [reply wdio.conf.js]: My bad.   Will fix everything before the class

**Comment by StanislavKim1** [comment]
@artsenius  Please when you have a chance could You please re-review my tests again. Thanks in advance.

**Pull request merged**